CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-20.1.2
+
+**Note**: The `linkerd-proxy` version included with this release is more
```

Comment:
We're back to a master proxy in this version.

Reply:
But we're not... yet. We will be, though.

Reply:
@olix0r @grampelberg Should add note that we are back to non-experimental under the Proxy changes section?